tests/test_api.py

```diff
@@ -140,6 +140,15 @@ def test_read_only(self, client, entities, endpoint, method, fields):
             assert (field not in body['results']
                     or body['results'][field] != 'test')
 
+    @pytest.mark.parametrize('endpoint', ENDPOINTS)
+    def test_predefined_kf_id(self, client, endpoint):
+        """ Check that posting predefined kf_id doesn't 500 """
+        resp = client.post(endpoint,
+                           data=json.dumps({'kf_id': 'XX_00000000'}),
+                           headers={'Content-Type': 'application/json'})
+        assert resp.status_code != 500
```

Comment:
Might just be better to check that the code is 200

Reply:
201?

Reply:
It will actually 400 because the required fks aren't there, so as long as we don't get 500, it should be ok.